test/fw/ptl/utils/plugins/ptl_test_runner.py

```diff
@@ -747,6 +755,80 @@ def __are_requirements_matching(param_dic=None, test=None):
             if not eff_tc_req['no_comm_on_mom']:
                 return False
 
+    def check_and_report_hardware_status(self):
```

Comment:
we are checking for core files as well. shall we name it check_core_files_and_disk_usage()?
if we are planning to add more hardware diagnostics to this function, we can keep "hardware" in the name, otherwise, all we are checking for is the disk usage.

Reply:
checks_core_files_and_hardware_status. 
        """
        function checks hardware status and core files
        every 5 minutes
        """

Reply:
instead of checks_ - it should be check_
are we going to add more diagnostics later on?

Reply:
yes